.github/workflows/contracts.yml

```
@@ -72,29 +72,29 @@ jobs:
         uses: nick-fields/retry@v3
         with:
           timeout_minutes: 15
           max_attempts: 3
           retry_wait_seconds: 15
-          command: yarn build
+          command: pnpm build
           working_directory: ./contracts
       - name: Run Tests (Contracts)
         working-directory: ./contracts
         # skip until they get fixed
         if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)